src/client/components/pages/Faculty/FacultyPage.tsx

```diff
@@ -138,9 +160,22 @@ const FacultySchedule: FunctionComponent = (): ReactElement => {
         )
         : (
           <>
+            <VerticalSpace>
+              <MenuFlexReverse>
+                <Checkbox
+                  id="showRetiredFaculty"
+                  name="showRetiredFaculty"
+                  label="Show Retired"
```

Comment:
The label here should be `Show "No Longer Active" Faculty`, as on [the existing course planner][1], since not everyone who's "No longer active" is necessarily retired.

[1]: https://planning.seas.harvard.edu/courses/app/#/faculty

Reply:
ok